### PR TITLE
Remove code from `CI.psm1` to optionally run Feature tests

### PR DESCRIPTION
**Pull request by RDIL**
# PR Summary
Makes feature tests run without the tag (which now causes issues)

## PR Context

@TravisEz13 said this needed to be done
## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [x] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

File: tools/ci.psm1 [hunk 1]
```diff
@@ -28,41 +28,7 @@ if($PSVersionTable.PSEdition -eq 'Desktop' -or $isWindows)
 # or is a pushed tag
 Function Test-DailyBuild
```
Comment (TravisEz13) [inline tools/ci.psm1]: There are some issues that occur when daily is true.  We need to fix that.  Linux and MacOS don't fail.  We need to review each case and make sure we do the right thing.

Comment (RDIL) [reply tools/ci.psm1]: Will do

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ To run the link-checker, follow these steps:
 
 ### Code Editor
 
-You should use the multi-platform [Visual Studio Code (VS Code)][use-vscode-editor].
+You should use the multi-platform [Visual Studio Code (VSCode)][use-vscode-editor].
```
Comment (iSazonov) [inline .github/CONTRIBUTING.md]: Why do we want to change this? `VS Code` is used on VS Code home site.

Comment (RDIL) [reply .github/CONTRIBUTING.md]: I'm pretty sure it is one word

Comment (vexx32) [reply .github/CONTRIBUTING.md]: Everywhere I see it, it's `VS Code` or `Visual Studio Code` 🤷‍♂️

Comment (RDIL) [reply .github/CONTRIBUTING.md]: reverted

**Comment by RDIL** [comment]
@TravisEz13 anything specific you need me to do right now?

**Comment by TravisEz13** [comment]
@RDIL ... I reviewed the usage of test-dailybuild and corrected the logic.

**Comment by RDIL** [comment]
Thanks

**Review by TravisEz13 — APPROVED** [review]

File: docs/testing-guidelines/testing-guidelines.md [hunk 3]
```diff
@@ -55,8 +55,6 @@ The Pester framework allows `Describe` blocks to be tagged, and our CI system re
 One of the following tags must be used:
 
 * `CI` - this tag indicates that the tests in the `Describe` block will be executed as part of the CI/PR process
-* `Feature` - tests with this tag will not be executed as part of the CI/PR process, but they will be executed on a daily basis as part of a `cron` driven build.
```
Comment (TravisEz13) [inline docs/testing-guidelines/testing-guidelines.md]: actually this should not be deleteed

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: docs/testing-guidelines/testing-guidelines.md [hunk 4]
```diff
@@ -55,9 +55,8 @@ The Pester framework allows `Describe` blocks to be tagged, and our CI system re
...
-  They indicate that the test will be validating more behavior, or will be using remote network resources (ex: package management tests)
```
Comment (TravisEz13) [inline docs/testing-guidelines/testing-guidelines.md]: This should still be a sub-bullet under feature

**Comment by ThreeFive-O** [comment]
@RDIL: Could you please update the PR title and give more details in the summary and context. Right now it could be interpreted that all Pester tests tagged with `Feature` will be removed, which is certainly not the case.
This PR rather only targets the [feature] tag in commit messages, which were necessary to include Pester tests marked with `Feature`.

**Comment by RDIL** [comment]
Sure

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@PoshChan 	Please remind me in 1 hour

**Comment by PoshChan** [comment]
@TravisEz13, all requests start with the magic word `Please`.

**Comment by RDIL** [comment]
@PoshChan please remind me in an hour

**Comment by RDIL** [comment]
@PoshChan please remind me in 1 hour

**Comment by PoshChan** [comment]
@RDIL, I do not understand: please remind me in an hour

**Comment by PoshChan** [comment]
@RDIL, will remind you in 1 hour

**Comment by PoshChan** [comment]
@RDIL, this is the reminder you requested 1 hour ago

**Comment by RDIL** [comment]
@TravisEz13 this is the reminder you requested an hour ago

**Pull request merged**